pom.xml

```diff
@@ -13,1025 +13,1025 @@
  **/
 -->
```

Comment:
❓ Can we switch back to 4-space indents and avoid the formatting changes? Pretty sure all of our pom (and most likely xml) files use 4-space indents.

Reply:
This is caused by the google format profile :-( It formats everything not just Java

Reply:
I agree that it would be nice not to include this change for now.

Reply:
I agree too. Didn't find an easy way in IntelliJ to address this. If I don't use the goggle profile, the other formatter ends up changing the files and then when the build runs on the command line, some lines gets corrupted sometimes.